pkgs/racket-doc/scribblings/style/textual.scrbl

```diff
@@ -144,10 +144,11 @@ read code on monitors that accommodate close to 250 columns, and on
 occasion, our monitors are even wider. It is time to allow for somewhat
 more width in exchange for meaningful identifiers.
 
-So, when you create a file, add a line with @litchar{;; } followed by ctrl-U 99 and
-@litchar{-}. When you separate "sections" of code in a file, insert the same line.
-These lines help both writers and readers to orient themselves in a file.
-In scribble use @litchar|{@; }| as the prefix.
+So, when you create a file, add a line with @litchar{;; } followed by ctrl-U 99
+and @litchar{-}. @margin-note*{In Vi, the command is 99a- followed by Esc.} When
+you separate "sections" of code in a file, insert the same line. These lines
+help both writers and readers to orient themselves in a file. In scribble use
```

Comment:
This is probably out of scope for the PR (so feel free to ignore this comment), but "scribble" probably should be capitalized (there might be other occurrences too?).

Reply:
I should at least fix new occurrences ;)

Reply:
Fixed.

<details><summary>range-diff</summary>

```
1:  945d895934 = 1:  35c19f1e83 docs: capitalize the noun Git
2:  d9943998ca ! 2:  6088cd1567 docs: mention the Vi command to add sections
    @@ pkgs/racket-doc/scribblings/style/textual.scrbl: read code on monitors that acco
     +So, when you create a file, add a line with @litchar{;; } followed by ctrl-U 99
     +and @litchar{-}. @margin-note*{In Vi, the command is 99a- followed by Esc.} When
     +you separate "sections" of code in a file, insert the same line. These lines
    -+help both writers and readers to orient themselves in a file. In scribble use
    ++help both writers and readers to orient themselves in a file. In Scribble use
     +@litchar|{@; }| as the prefix.
      
      @; -----------------------------------------------------------------------------
3:  01b511b849 = 3:  1e7b35da0a docs: link fx+ and unsafe-fx+
4:  f12b1170c5 = 4:  c3e32a5afa docs: correct Git pull command
5:  78968271a5 = 5:  372bbd4ad5 docs: unquote "merge commit"
6:  8cf1dc6855 = 6:  e48525eeb7 docs: correct macro body
7:  090fef892e = 7:  38b3c0a75e docs: make explicit the convention for Latin
```

</details>